### ISSUE TITLE
Pure ODD minOccurs maxOccurs not respected in RNG

### ISSUE THREAD

**Issue by jamescummings** [TEIC/Stylesheets#174]
If I create a new element with a content model of:

```
<content><elementRef key="name" minOccurs="5" maxOccurs="7"/></content>

```

(for example, but any numerical range) then this should mean that I must have 5, 6, or 7 'name' elements. What happens in the RNG is that it requires me to have 5 (fine), but complains at 6 or 7 instances of 'name' elements.

Clearly this is an artificial example.  Am I misunderstanding the attributes, or should this work?

**Comment by lb42** [comment]
This is a known bug.

**Comment by jamescummings** [comment]
Closed. Fixed with #135
